src/main/resources/db/migration/common/V1_375__update_automated_tasks_default_values.sql

```diff
@@ -0,0 +1,75 @@
+-- Run every hour at 10 mins past the hour between 8pm and 8am with batch size 5,000
```

Comment:
@cmagowan-hmcts @jackmaloney this adds another flyway script, it doesn't change the table definitions, but not sure if the migration team need to be aware?

Reply:
Yeh, they're already aware.